rules-reviewed/jboss-eap/websphere/tests/WebSphereMQ.windup.test.xml

```diff
@@ -0,0 +1,322 @@
+<?xml version="1.0"?>
+<ruletest xmlns="http://windup.jboss.org/schema/jboss-ruleset">
+    <testDataPath>samples/WebSphereMQ</testDataPath>
+    <rulePath>../WebSphereMQ.windup.xml</rulePath>
+    <ruleset>
+        <metadata></metadata>
+        <rules>
+            <rule>
```

Comment:
I like ruleset-id-02000-test-n personally. This will also show up in the failure report if there are test failures, which should make finding the offending test-case (and expected original rule) easier.